### PR TITLE
TypeScript conversion batch 2

### PR DESCRIPTION
**Pull request by drosenberg123**
- updated second batch js->ts
- fixed babel preset misconfiguration
- set ts config target es5->es6

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -30,7 +30,21 @@ const config = {
...
+                    modules: false,
+                    targets: {
+                      chrome: '88',
+                      ie: '11',
+                      edge: '88',
+                      firefox: '79',
+                      safari: '13.1.3',
+                    },
+                  }
+                ],
```
Comment (drosenberg123) [inline webpack.config.js:45]: - `@babel/env` was mistakenly added, but not used; replaced with `@babel/preset-env`

File: src/RoleObjectFactory.ts [hunk 2]
```diff
@@ -53,30 +55,42 @@ import TreeItemData from './RoleObjects/TreeItemData';
 import ToolBarData from './RoleObjects/ToolBarData';
 import UnorderedListData from './RoleObjects/UnorderedListData';
 
+type EventHandler = (evt) => void;
```
Comment (tussharrane) [inline src/RoleObjectFactory.ts:58]: Missing type for `evt`
```suggestion
type EventHandler = (evt: Event) => void;
```

File: src/RoleObjects/AccessibilityObject.ts [hunk 3]
```diff
@@ -7,20 +7,23 @@ export type AccessibleDisplayObject = DisplayObject & {
   accessible?: AccessibilityObject;
 };
 
-type ReactProps = {
+type EventHandler = (evt) => void;
```
Comment (tussharrane) [inline src/RoleObjects/AccessibilityObject.ts:10]: add a type to the `evt`

**Review by mmoss — APPROVED** [review]
Nice work!

**Review by mmoss — APPROVED** [review]
:shipit:

**Pull request merged**